packages/superset-ui-chart-controls/src/utils/titleOptions.ts

```diff
@@ -0,0 +1,14 @@
+export const TITLE_MARGIN_OPTIONS: [number, number][] = [
```

Comment:
These constants, if used only in `chartTitle.tsx` component, suggest defining within the component.

Reply:
Done